manifest.toml

```
@@ -4,23 +4,21 @@ packaging_format = 2
 
 id = "motioneye"
 name = "motionEye"
-description.en = "Explain in *a few (10~15) words* the purpose of the app or what it actually does (it is meant to give a rough idea to users browsing a catalog of 100+ apps)"
-description.fr = "Expliquez en *quelques* (10~15) mots l'utilité de l'app ou ce qu'elle fait (l'objectif est de donner une idée grossière pour des utilisateurs qui naviguent dans un catalogue de 100+ apps)"
+description.en = "motionEye is a web frontend for the motion daemon, written in Python."
+description.fr = "motionEye is a web frontend for the motion daemon, written in Python"
 
-version = "1.0~ynh1"
+version = "ae136a4c45034839db10388dd7cd5ad7ecfe2f21.2023.07.31"
```

I guess the version number is wrong? I would use this:
```suggestion
version = "0.2023.07.31"
```